website/lab/02/index.mdx

```diff
@@ -122,10 +140,90 @@ The name *Embassy* is derived from **Emb**edded **Asy**nchronous Rust.
 
 The Embassy Framework is a collection of crates. Building an *empty* firmware that uses
 embassy requires:
-- adding the Embassy HAL Implementation for a specific board, in this case RP2350;
+- adding the Embassy HAL Implementation for a specific board, in this case STM32U545 or RP2350;
 - adding the core Embassy crates, that provide the executor, timers and futures;
 - adding the `cortex-m-rt` and `defmt` crates that Embassy requires.
 
+<Tabs>
+  <TabItem value="stm32u5" label="STM32 Nucleo-U545RE-Q" default>
+    ```toml
+# STM32U545 HAL
+embassy-stm32 = { version = "0.2.0", git = "https://github.com/embassy-rs/embassy", rev = "2e7a2b6", features = [
```

Comment:
We should use the new version of embassy, can you try the latest GitHub release?

Reply:
Can we try the latest release version of embassy https://github.com/embassy-rs/embassy/tree/embassy-stm32-v0.4.0 (tag `stm32-v0.4.0`)?